com.claudiocambra.brickbuster.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id": "com.claudiocambra.brickbuster",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "modules": [
+        {
+            "name": "BrickBuster",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D brickbuster_linux.x86_64 /app/bin/brickbuster_linux.x86_64",
+                "install -D com.claudiocambra.brickbuster.desktop /app/share/applications/com.claudiocambra.brickbuster.desktop",
+                "install -D com.claudiocambra.brickbuster.png /app/share/icons/hicolor/512x512/apps/com.claudiocambra.brickbuster.png",
+                "install -D com.claudiocambra.brickbuster.appdata.xml /app/share/appdata/com.claudiocambra.brickbuster.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/claucambra/BrickBuster/releases/download/1.0/brickbuster_linux.x86_64",
+                    "tag": "1.0",
+                    "sha256": "dbca7ed60428ef911847447035589a7581d71e30c99c1910a19fac95dce16550"
+                },
+                {
+                    "type": "file",
+                    "path": "com.claudiocambra.brickbuster.desktop",
+                    "dest-filename": "com.claudiocambra.brickbuster.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.claudiocambra.brickbuster.png",
+                    "dest-filename": "com.claudiocambra.brickbuster.png"
+                },
+                {
+                    "type": "file",
+                    "path": "com.claudiocambra.brickbuster.appdata.xml",
+                    "dest-filename": "com.claudiocambra.brickbuster.appdata.xml"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
we usually like to have this toward the top. Not a breaking issue, just common use.

Reply:
Noted! Moved it to the top